lib/results.js

```diff
@@ -157,8 +157,10 @@ function encodeResult (res, count) {
     if (res.at) {
         output += inner + 'at: ' + res.at + '\n';
     }
-    if (res.operator === 'error' && res.actual && res.actual.stack) {
-        var lines = String(res.actual.stack).split('\n');
+    var error = (res.actual instanceof Error ? res.actual : res.error)
```

Comment:
`instanceof` will fail on cross-realm Error instances (like from an iframe, or the `vm` module.